server/views/pages/contacts/manage/contactDetails/manageIdentityNumbers.njk

```diff
@@ -4,51 +4,49 @@
 
 {% if contact.identities and contact.identities.length > 0 %}
     {% for item in contact.identities %}
-        {% set identityNumbers = (
-            identityNumbers.push(
-                {
-                    key: {
-                        text: (item.identityTypeDescription | capitalizeFirstLetter)
-                    },
-                    value: {
-                        html: item.identityValue + ('<p>' + item.issuingAuthority + '</p>' if item.identityType == 'PASSPORT' else ''),
-                        classes: 'confirm-' + item.identityType + '-value'
-                    },
-                    actions: {
-                        items: [
-                            {
-                                href: "/prisoner/" + prisonerDetails.prisonerNumber + "/contacts/manage/" + contact.id + "/identity/" + item.contactIdentityId + "/edit",
-                                text: "Edit",
-                                visuallyHiddenText: "identity number",
-                                attributes: {"data-qa": "edit-identity-number-" + item.contactIdentityId},
-                                classes: "govuk-link--no-visited-state"
-                            },
-                            {
-                                href: "/prisoner/" + prisonerDetails.prisonerNumber + "/contacts/manage/" + contact.id + "/identity/" + item.contactIdentityId + "/delete",
-                                text: "Delete",
-                                visuallyHiddenText: "identity number",
-                                attributes: {"data-qa": "delete-identity-number-" + item.contactIdentityId},
-                                classes: "govuk-link--no-visited-state"
-                            }
-                        ]
-                    }
-                }
-        ), identityNumbers) %}
+        {% set actions = [] %}
+
+        {% if item.identityTypeIsActive == true %}
+            {% set actions = actions.concat([{
```

Comment:
what's the change from push to concat do? it looks like you're only adding 1 at a time if concat is adding 2 arrays?